### ISSUE TITLE
Quota increase option for Custom domain

### ISSUE THREAD

**Issue by soyhanbeyazit** [aws/apprunner-roadmap#249]
**Community Note**
* Please vote on this issue by adding a 👍 reaction to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do * not help prioritize the request
If you are interested in working on this issue or have submitted a pull request, please leave a comment

**Tell us about your request**
What do you want us to build?

**Describe alternatives you've considered**
At the moment, the custom domain limit is 5. It would be nice to have quota request for this limitation so that it can be increase by request. 
This is a limitation for our whitelabel solution on our SaaS app

**Additional context**

**Attachments**

**Comment by TreyWW** [comment]
Hey @soyhanbeyazit, great feature request! Did you manage to find a solution for your application? Could you use something like Cloudfront as an almost proxy to App Runner and then use multiple domains using that Cloudfront URL?

Appreciate it!

**Comment by soyhanbeyazit** [comment]
Well, for now, the app landed on Elasticbeanstalk, but your suggestions seems interesting, I will see if it would simplfiy the setup.

**Comment by TreyWW** [comment]
Yeah. I was searching about how to get logs for every request and someone suggested to use CF in front and then that way you can also get logs, caching and multiple domains. Worth checking out! I haven't tried myself yet though, next on this list! Good luck though
